agent/tool-scripts/postprocess/unittests

```diff
@@ -2,6 +2,7 @@
 export LANG=C.UTF-8
 export LC_ALL=C.UTF-8
 export PERL_HASH_SEED=0
+let MAX_PERFTEST_SECONDS=300  # 5 minutes
```

Comment:
Do we really want a single fixed performance threshold for all performance tests? Another option would be to define a new setup file (like the server's `state` directory protocol) that allows per-test configuration. E.g., for tests requiring database pre-loads, I created a `${test}.db` file that would be sourced. Something like a `${test}.perf` could not only set per-test timeouts, but could also be used to identify a performance test instead of requiring a specific name prefix.

Then again, if you think a global fixed constant is good enough for now, there's enough in this PR and I'm not going to object.

Reply:
Yeah, the problem is, where to draw the line....  I think we're all hoping that this entire framework is going to go away and be replaced by something better (e.g., something written in Python).  Moreover, if a performance test fails to complete by the deadline, it should be killed and the rest of the tests allowed to continue, but I deemed that to be beyond the scope of this change, as well.  And, ideally, the threshold would be not only per-test but also adaptive -- "I, Mr. Jenkins, think this test failed because the current performance is out of line with recent runs of this test."

So, your suggestion is an excellent one, but it's just a little more than I'm inclined to take on for this PR.